### PR TITLE
Html5 playback in bsp

### PR DESCRIPTION
**Pull request by JaKraken**
Creating a new playback strategy for HMTL5 players. along with live modifiers

Brought in from TAL. Hybrid strategy is set up to use this new native strategy

Unit tests coming soon!

### CODE REVIEW

File: docs/example-app/script/index.js [hunk 1]
```diff
@@ -130,6 +130,10 @@ define([
     var liveSupport = 'seekable';
     var enableSubtitles = false;
 
+    window.bigscreenPlayer = {
+      playbackStrategy : 'nativehtml5strategy'
```
Comment (Luke-Chatburn) [inline docs/example-app/script/index.js]: Is this not suppose to be nativestrategy?

File: script/parsers/manifestparser.js [hunk 2]
```diff
@@ -29,18 +31,23 @@ define('bigscreenplayer/parsers/manifestparser',
          } else {
            return {error: 'Error parsing DASH manifest attributes'};
          }
+         DebugTool.info('Done parsing MPD');
```
Comment (Luke-Chatburn) [inline script/parsers/manifestparser.js]: A lot of these logs you should probably remove... errors are fine to leave in. I am assuming you just used them for testing purposes?

File: script/playbackstrategy/modifiers/live/seekable.js [hunk 3]
```diff
@@ -0,0 +1,147 @@
+/**
+ * @fileOverview Requirejs module containing device modifier for live playback
+ * with support level Seekable
+ * @preserve Copyright (c) 2013-present British Broadcasting Corporation. All rights reserved.
+ * @license See https://github.com/bbc/tal/blob/master/LICENSE for full licence
```
Comment (Luke-Chatburn) [inline script/playbackstrategy/modifiers/live/seekable.js]: Does all this stuff need removing?

File: docs/example-app/script/index.js (hunk 1 above)
Comment (magnuswoody) [reply docs/example-app/script/index.js]: Reverted it back to MSE strategy (was used for native testing purposes). The intention was to keep the example-app MSE.

File: script/parsers/manifestparser.js (hunk 2 above)
Comment (magnuswoody) [reply script/parsers/manifestparser.js]: Removed the debug tool require and extra logging statements.

File: script/playbackstrategy/modifiers/live/seekable.js (hunk 3 above)
Comment (magnuswoody) [reply script/playbackstrategy/modifiers/live/seekable.js]: Removed with recent [commit](https://github.com/bbc/bigscreen-player/pull/29/commits/68f349789e7c0bc00dffb35ea07b722a8f8518c4)

File: docs/example-app/script/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (magnuswoody) [reply docs/example-app/script/index.js]: Should be sorted in the latest [commit](https://github.com/bbc/bigscreen-player/pull/29/commits/d924dac5f6b9ac39c9d0f2d316b4882e5b058d1f) 👍

File: spec/support/boot.js [hunk 4]
```diff
@@ -74,11 +74,7 @@ WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
   var filterSpecs = !!queryString.getParam('spec');
 
   // Overrides
-  env.stopOnSpecFailure(true);
```
Comment (jlks) [inline spec/support/boot.js]: These need putting back as they don't work from the grunt plugin config even without the boot.js

File: gruntfile.js [hunk 5]
```diff
@@ -12,6 +12,10 @@ module.exports = function (grunt) {
       ],
       template: 'spec/support/test-runner/SpecRunner.html',
       outfile: 'spec/support/test-runner/WebRunner.html',
+      failFast: true,
```
Comment (jlks) [inline gruntfile.js]: These don't work. run `npm run spec` and the options are not applied to the web runner. See the comment below on boot.js

**Pull request merged**